packages/models/proto/v2/eservice/events.proto

```diff
@@ -20,7 +20,7 @@ message EServiceDeletedV2 {
 message EServiceClonedV2 {
   EServiceV2 sourceEservice = 1;
   string sourceDescriptorId = 2;
-  EServiceV2 clonedEservice = 3;
+  EServiceV2 eservice = 3;
 }
```

Comment:
@galales I also updated EServiceClonedV2 as discussed [here](https://github.com/pagopa/interop-be-monorepo/pull/245#discussion_r1512481832).